.dev/.env.dist

```diff
@@ -0,0 +1,27 @@
+RAILS_ENV=development
+REDIS_URL=redis://localhost:6379
+
+# Zammad configuration.
+
+ZAMMAD_BIND_IP=0.0.0.0
+DB_ADAPTER=postgresql
+DB_PASSWORD=zammad
+REDIS_URL=redis://redis:6379
+ELASTICSEARCH_URL=http://elasticsearch:9200
+DB_ADAPTER=postgresql
+DB_HOST=zammad-postgresql
+DB_NAME=zammad
+DB_USER=zammad
+DB_PASSWORD=zammad
+
+# PostgreSQL configuration.
+
+POSTGRESQL_PASS=zammad
+POSTGRESQL_USER=zammad
+POSTGRESQL_HOST=zammad-postgresql
+POSTGRESQL_PORT=5432
+POSTGRESQL_DB=zammad
+POSTGRESQL_OPTIONS=
+
+# Elasticsearch configuration.
+ELASTICSEARCH_HOST=elasticsearch
```

Comment:
Perhaps another thought about this .env file. I'm not sure if it is really needed, as everything is really internal to the stack.
Another idea might be to move these settings to a shared, static definition in the .yml file like we do in zammad-docker-compose which is used by the services. Perhaps at least for those settings where a modification does not make sense (such as ES hostname). But perhaps for all?
Having the ability to pass in custom stuff for the local developer's stack in addition to that might still be a good idea.
Just an idea, though.